targets/TARGET_NUVOTON/TARGET_M2351/crypto/crypto-misc.cpp

```diff
@@ -19,33 +19,51 @@
 #include "mbed_assert.h"
 #include "mbed_critical.h"
 #include "mbed_error.h"
+#if MBED_CONF_RTOS_PRESENT
+#include "cmsis_os2.h"
```

Comment:
Is this really needed? You're not using it directly, but maybe something's missing from another header?

Reply:
@kjbracey-arm I removed them.